components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -140,6 +143,21 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
             this.toDispose.push({
                 dispose: () => window.removeEventListener("message", setStateEventListener),
             });
+            // TODO(hw): end of delete
+
+            this.ideFrontendService = getIDEFrontendService(this.props.workspaceId, sessionId, getGitpodService());
```

Comment:
Could it happen that this code is called twice for some reasons?

Reply:
This is a singleton so it's fine